harness/determined/common/util.py

```diff
@@ -210,6 +210,28 @@ def parse_protobuf_timestamp(ts: str) -> datetime.datetime:
     return datetime.datetime.fromisoformat(ts)
 
 
+def is_protobuf_timestamp(ts: str) -> bool:
+    """
```

Comment:
suggestion: Could you please format this docstring per https://google.github.io/styleguide/pyguide.html#383-functions-and-methods?

We've been inconsistent within the harness up until this point, and if you prefer this can be one of the docstrings we edit when at the end of the project we go through and standardize. Up to you.